taskcluster/__init__.py

```diff
@@ -14,6 +11,3 @@
     if len(log.handlers) == 0:
         log.addHandler(logging.StreamHandler())
 log.addHandler(logging.NullHandler())
-
-from client import *  # NOQA
```

Comment:
Does this mean we now need to do

`import taskcluster.client`?

Instead of `import taskcluster`  ?  I'm not too familiar with the new import system and I'm on my phone right now.